src/runtime/pay-flow.js

```diff
@@ -271,6 +271,10 @@ export class PayCompleteFlow {
    */
   complete() {
     this.eventManager_.logSwgEvent(AnalyticsEvent.ACTION_ACCOUNT_CREATED, true);
```

Comment:
nit: The spacing here seems off. Could you reformat? Although I don't know if this is due to github or the actual code spacing.